ProcessMaker/Http/Middleware/GenerateMenus.php

```diff
@@ -109,23 +115,8 @@ public function handle(Request $request, Closure $next)
                 ]);
             }
         });
-        Menu::make('sidebar_task', function ($menu) {
```

Comment:
Please not remove this because gives an error:
`Call to a member function first() on null`
some packages use the following code
`$menu = Menu::get('sidebar_task')->first();`